github-workflow-roles/main.tf

```
@@ -42,6 +44,14 @@ locals {
     include_org_condition     = true
   }
 
+  path = var.principal_type == "breakglass" ? "/breakglass/" : "/RVM/"
```

```suggestion
  path = var.principal_type == "breakglass" ? "/RVM/breakglass/" : "/RVM/"
```

This allows us to keep the existing permissions, which specify the first key in the path be `RVM`

---

I don't want the workflow role to be able to assume the break glass role, that is why I put them in two different paths.